src/StatsdClient/StatsdClient.csproj

```diff
@@ -22,5 +23,10 @@
       <Version>4.3.0</Version>
     </PackageReference>
   </ItemGroup>
-
+  <ItemGroup>
```

Comment:
Can we get to run this stuff run in the CI?

Reply:
With the changes done in `src/StatsdClient/StatsdClient.csproj` warnings are generated when compiling.

We should catch StyleCop warnings in AppVeyor as I used `/warnaserror` flag at: https://github.com/DataDog/dogstatsd-csharp-client/pull/111/files#diff-180360612c6b8c4ed830919bbb4dd459R10.

I will test it once previous PRs will be merged (AppVeyor tests are failing).

Reply:
Good catch!

There is an existing issue that prevents `/warnaserror` to work as expected: https://github.com/dotnet/installer/issues/1708.

Reply:
Fixed.
This build https://ci.appveyor.com/project/Datadog/dogstatsd-csharp-client/builds/32467628 is an example of Style Cop issues caught by the CI.